pkg/controller/servicemesh/webhooks/mutation/controlplane.go

```diff
@@ -93,6 +93,25 @@ func (v *ControlPlaneMutator) Handle(ctx context.Context, req admission.Request)
 		}
 	}
 
+	// removing tracing by default on creation of v2.6 SMCP
+	if req.AdmissionRequest.Operation == admissionv1beta1.Create {
+		newOpenShiftTracing := mutator.IsTracingTypeSpecified()
+
+		if !newOpenShiftTracing {
+			var ver versions.Version
+			var err error
+			// If version is not specified
+			if currentVersion == "" {
+				ver, err = versions.ParseVersion(mutator.DefaultVersion())
+			} else {
+				ver, err = versions.ParseVersion(currentVersion)
+			}
```

Comment:
Instead of copying and pasting this piece of code, please refactor it as I did it [here](https://github.com/maistra/istio-operator/compare/maistra-2.6...jewertow:istio-operator:OSSM-6693-version-3-backup#diff-0b39680bd1f117e8386c7dd7be4e9b8ff69867f3df17df13037795272ae90f49).